### PR TITLE
Change background color in nested sub table

### PR DESCRIPTION
**Pull request by is-ashish**
1. Change background color inside nested table -:
2. Add border in nested table header

Screenshot from 2020-10-30 12-35-35

Signed-off-by: ashishjain <ashish.jain@mayadata.io>

### CODE REVIEW

File: scss/product/components/_tables.scss [hunk 1]
```diff
@@ -226,27 +226,25 @@ table tr:nth-child(even) td{
...
       }
-      tbody {
-        td {
-          font-size: 16px;
-          line-height: 24px;
-        }
+    }
+    tbody {
+      td {
+        font-size: 16px;
```
Comment (rakeshyc) [inline scss/product/components/_tables.scss:245]: lgtm

**Review by rakeshyc — APPROVED** [review]
lgtm

**Comment by Pallavi-PH** [comment]
> Change background color inside nested table -:
> Screenshot from 2020-10-29 20-33-52
> 
> Signed-off-by: ashishjain [ashish.jain@mayadata.io](mailto:ashish.jain@mayadata.io)

I think border-bottom for the header is missing

**Comment by is-ashish** [comment]
Addressed changes @Pallavi-PH PTAL.

**Review by Pallavi-PH — APPROVED** [review]
LGTM

**Pull request merged**